index.html

```diff
@@ -1,12 +1,66 @@
 <!DOCTYPE html>
 <html lang="en">
 <head>
+    <link rel="stylesheet" href="styles/index.css">
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Weather Report</title>
 </head>
 <body>
-    
+    <header>
+        <h1> The Weather Report </h1>
+    </header>
+    <main>
+        <section>
+            <h2 id="amazingCity"></h2>
+        </section>
+        <section id="temp">
+            <p> Temperature in °F</p>
```

Comment:
This _is_ the title of this part of the page and so should have been a `h2`:

```suggestion
            <h2> Temperature in °F</h2>
```

If the `h2` isn't the right size you can always change the default styles with CSS.